app/components/action_button.rb

```diff
@@ -1,14 +1,14 @@
 # frozen_string_literal: true
 
-# Draws a blue button in the side menu
+# Draws a red button on the item detail page
 class ActionButton < ApplicationComponent
   # @param [Hash] properties the button properties
-  def initialize(label:, url:, method: nil, confirm: nil, new_page: nil, check_url: nil, disabled: nil)
+  def initialize(label:, url:, method: nil, confirm: nil, open_modal: false, check_url: nil, disabled: nil)
```

Comment:
didn't this just flip the default?  If they specified nothing before, it would open a modal.  Now if you specify nothing it doesn't open a modal.  This seems like a change that could inadvertently cause problems.  Can you move it to a separate PR?

Reply:
well yes it flips it, but makes the intent clearer to me (I wasted an hour trying to figure out why my POST request wasn't working for the new button)

could move to a new PR, but will then have to wait for this one to merge to move forward

Reply:
I think it's fine as long as you touched all the callers to ensure the right behavior?

Reply:
I  fixed all  of the callers (they were all in the `ControlsComponent`, which also had other changes as part of this PR anyway)

Reply:
What about [app/views/items/_button_link_list.html.erb](https://github.com/sul-dlss/argo/blob/f728b0922975739ede99bfcb86d7cf82d611d209/app/views/items/_button_link_list.html.erb) ?

Reply:
Okay 👍